src/store/item-store.js

```diff
@@ -1,26 +1,23 @@
 import { Store } from 'consus-core/flux';
-
+import { searchItem } from '../lib/api-client.js';
 let item = null;
 
 class ItemStore extends Store {
     getItem() {
-        if (item === null) {
-            return item;
-        }
-        return {
-            id: item.id,
-            status: item.status
-        };
+        return item;
+    }
+    getAllItems(){
+        return item;
+    }
+    searchItemByAddress(address){
```

Comment:
Stores probably shouldn't be returning async promises. Is there any reason for this method, when we can just call `searchItem` from the `api-client`?